tests/foreman/sys/test_hot_backup.py

```diff
@@ -423,8 +422,7 @@ def test_negative_incremental_with_no_src_directory(self):
                 'katello-backup --incremental',
                 output_format='plain'
             )
-            self.assertEqual(result.return_code, 1)
-            self.assertIn(NOPREV_MSG, result.stderr)
+            self.assertEqual(result.return_code, 255)
```

Comment:
This better to assert that return_code != 0 and let the error message as per doc string expectedresults, what do you think